test/e2e/framework/util.go

```diff
@@ -5130,3 +5130,16 @@ func CreateFileForGoBinData(gobindataPath, outputFilename string) error {
 	}
 	return nil
 }
+
+func CleanupGCEResources(loadBalancerName string) (err error) {
+	gceCloud, ok := TestContext.CloudConfig.Provider.(*gcecloud.GCECloud)
+	if !ok {
+		return fmt.Errorf("failed to convert CloudConfig.Provider to GCECloud: %#v", TestContext.CloudConfig.Provider)
+	}
+	gceCloud.DeleteFirewall(loadBalancerName)
+	gceCloud.DeleteGlobalForwardingRule(loadBalancerName)
```

Comment:
I tend to delete the forwarding rules, seems that method was not exposed as public, I will create public wrapper in gce.go